RELEASING.md

```diff
@@ -259,36 +276,33 @@ as described in the section above.
    make release-publish
    ```
 
-1. Merge the PR. This will cause candidate.docs.projectcalico.org to be updated (after a few minutes). Validate that everything looks correct before proceeding to the next step.
+1. Merge the PR. This will cause docs.projectcalico.org to be updated (after a few minutes). Validate that everything looks correct.
 
-1. Checkout the master branch
+## Adding the previous release to docs.projectcalico.org/archive
 
-1. In [netlify.toml](netlify.toml):
-
-   1. Set the `CANDIDATE_RELEASE` environment variable back to an empty string.
-
-   1. Update the `CURRENT_RELEASE` environment variable.
+1. Enable branch deployments for previous release branch on netlify by adding branch name to `Deploy contexts` in site settings.
+ 
+1. Checkout to previous release branch. Add a commit to the branch, an empty commit if necessary (to trigger the branch deploy). 
```

Comment:
I really wish this wasn't required but in your testing you couldn't find another way to trigger the new branch build, right?

Reply:
yeah.. I couldn't.. there are [build hooks](https://docs.netlify.com/configure-builds/build-hooks/#parameters) but they only work for branches that are deployed at least once